kernel/byterun/coq_interp.c

```diff
@@ -203,11 +205,13 @@ if (sp - num_args < coq_stack_threshold) {                                     \
     *sp = swap_accu_sp_tmp__;                   \
   }while(0)
 
+#if OCAML_VERSION < 41000
 /* For signal handling, we hijack some code from the caml runtime */
 
 extern intnat caml_signals_are_pending;
 extern intnat caml_pending_signals[];
```

Comment:
Both constants are qualified upstream [at least in 4.07] as `extern intnat volatile`, thus this must be fixed to allow the 4.07 build I think.

Reply:
Seem my C compiler is more strict than the one in the CI.

Reply:
You more strict compiler is right.